### DISCUSSION TITLE
May I add my algorithm for filling matrix by spiral?

### DISCUSSION THREAD

**Discussion by yan-aint-nickname** [TheAlgorithms/Go#645] [Show and tell]
I wrote an algorithm and saved it to [gist](https://gist.github.com/yan-aint-nickname/e0f856022d1f09193e126d1777a24796)
Is it worth it to add to this repo?

**Comment by siriak** [comment]
Hi, it's not a computer science algorithm, but a coding challenge. We don't add such problems.

**Reply by yan-aint-nickname** [reply]
Ok, thank you for your answer!
